views/js/controller/Diagnostic/index.js

```diff
@@ -118,7 +118,7 @@ define([
             var extension = $container.data('extension') || 'taoClientDiagnostic';
             var $list = $container.find('.list');
             var dataset = $container.data('set');
-            var config = $container.data('config') || {};
+            var config = $container.data('config')['diagnostic'] || {};
```

Comment:
Could not it be managed by the backend instead?

Reply:
No idea how at the moment.